test/component/payment/testPaymentBreakdown.js

```diff
@@ -60,6 +60,50 @@ describe('payment-breakdown', () => {
                 });
         });
 
+        it('test total=0.00 Pay and submit application on button Label', (done) => {
```

Comment:
surely you mean
test total>0.00 Pay and Submit....
?

Reply:
thanks, fixed the typo